Doc/library/inspect.rst

```diff
@@ -354,6 +357,10 @@ attributes (see :ref:`import-mod-attrs` for module attributes):
       Functions wrapped in :func:`functools.partial` now return ``True`` if the
       wrapped function is a :term:`coroutine function`.
 
+   .. versionchanged:: 3.12
+      Functions wrapped in :func:`functools.partialpartial` now return ``True``
```

Comment:
```suggestion
      Functions wrapped in :func:`functools.partialmethod` now return ``True``
```

Reply:
🤦‍♂️ Good catch, that's a really dumb merge error on my part!